src/views/ImportFlow.vue

```diff
@@ -218,9 +218,23 @@ class ImportFlow extends mixins(Lang, Routes) {
       persistRoute: this.route('flows.persistFlowImport', {}),
       flowContainer: this.flowContainer,
     })
+    const numberOfFlows = get(flowContainer, 'flows.length', 0);
```

Comment:
Maybe let's avoid `get` and use a dot chaining instead, followed by `?? defaultValue` if needed.